File: fix_hero_text.pl
use strict;
use warnings;

my $file = 'src/pages/Home.tsx';
open my $fh, '<', $file or die $!;
my $content = do { local $/; <$fh> };
close $fh;

if ($content =~ s/(<section className="relative w-full min-h-screen.*?)(?=\{\/\* C\. The Ticker)/
    my $c = $1;
    $c =~ s!text-text-main!text-white!g;
    $c =~ s!text-text-muted!text-white\/90!g;
    $c;
/es) { print "Updated Hero Text\n"; }

open my $out, '>', $file or die;
print $out $content;
close $out;

File: fix_hero_text.py
import re

with open('src/components/Hero.tsx', 'r') as f:
    hero = f.read()

hero = hero.replace('text-white font-sans', 'text-[#c6f6d5] font-sans')
hero = hero.replace('text-white max-w-5xl', 'text-[#c6f6d5] max-w-5xl')
hero = hero.replace('text-white/90 font-sans', 'text-[#c6f6d5]/90 font-sans')

with open('src/components/Hero.tsx', 'w') as f:
    f.write(hero)

print("Updated Hero text color")
